js/reference.js

```diff
@@ -1754,13 +1754,14 @@
                  *
                  * github issue: #425
                  */
-
+                var self = this, delFlag = 'DEL';
```

Comment:
Instead of just defining this string here. Define an object the same way we have contexts defined in the utilities.js file. I'm not sure what is the best way to name that object. But eventually it should include `update` and `create`  too.

```javascript
module._operations = Object.freeze({
  DELETE: "delete",
  CREATE: "create",
  UPDATE: "update",
  READ: "read"
});
```